authui/eslint.config.mjs

```diff
@@ -0,0 +1,56 @@
+import oursky from "@oursky/eslint-plugin";
+import compat from "eslint-plugin-compat";
+import globals from "globals";
+
+// FIXME(eslint): For now we only lint src/authflowv2.
+// but we should lint everything in src.
+const js = "src/authflowv2/**/*.{js,jsx,mjs,mjsx}";
+const ts = "src/authflowv2/**/*.{ts,tsx,mts,mtsx}";
```

Comment:
It this because there are too much errors in other parts?

Reply:
No, originally eslint lints authflowv2 files only. See the deleted eslintignore file.